string name
float32 effort
float32 velocity
float32 position
string note
float32 temp_mos
float32 temp_rotor
